CHANGELOG/3.4.1/3.4.1.md

```diff
@@ -0,0 +1,65 @@
+# 3.4.1.0
+
+## Kong
+
+
+### Additions
+
+#### Core
+
+- Support HTTP query parameters in expression routes.
+  [#11348](https://github.com/Kong/kong/pull/11348)
+
+
+### Dependencies
+
+#### Core
+
+- Fix incorrect LuaJIT LDP/STP fusion on ARM64 which may sometimes cause incorrect logic
+  [#11537](https://github.com/Kong/kong-ee/issues/11537)
+  [KAG-2473](https://konghq.atlassian.net/browse/KAG-2473)
+
+
+
+### Fixes
+
+#### Core
+
+- Removed a hardcoded proxy-wasm isolation level setting that was preventing the
+  `nginx_http_proxy_wasm_isolation` configuration value from taking effect.
+  [#11407](https://github.com/Kong/kong/pull/11407)
+- Fix an issue that the TTL of the key-auth plugin didnt work in DB-less and Hybrid mode.
+  [#11464](https://github.com/Kong/kong-ee/issues/11464)
+  [FTI-4512](https://konghq.atlassian.net/browse/FTI-4512)
+- Fix a problem that abnormal socket connection will be reused when querying Postgres database.
+  [#11480](https://github.com/Kong/kong-ee/issues/11480)
+  [FTI-5322](https://konghq.atlassian.net/browse/FTI-5322)
+- Fix upstream ssl failure when plugins use response handler
+  [#11502](https://github.com/Kong/kong-ee/issues/11502)
+  [FTI-5347](https://konghq.atlassian.net/browse/FTI-5347)
```

Comment:
Should we add FTI or KAG link in CE's change log?

Reply:
Please discuss this with @vm-001 for changelog optimization later. For now, let's keep this to unblock 3.4.1 release and I think the FTI/KAG will not be rendered on the user-facing HTML page.